### ISSUE TITLE
Spelling / missing word in sentence

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#504]
```
Please describe the issue:

In the page - "Offline": What does it mean...

In the section titled Web Storage, paragraph 2: the first sentence doesn't make 
sense (probably a word and space character missing).

Quote:
"Although the spec indicates values can be any type, this is not the case in 
practiceare presently supporting only string values, so you will need to 
serialize your values for storage (i.e., convert them to strings). "
```

Original issue reported on code.google.com by `jones.no...@gmail.com` on 11 Apr 2011 at 3:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 11 Apr 2011 at 4:31
- Changed state: **Assigned**
- Added labels: **Milestone-5.5**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 11 Apr 2011 at 4:32
- Added labels: **Milestone-6**
- Removed labels: **Milestone-5.5**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7f2b4625d2f9.
```

Original comment by `ericbide...@html5rocks.com` on 1 Jun 2011 at 11:44
- Changed state: **Fixed**
